kano_updater/ui/install_window.py

```diff
@@ -45,6 +50,9 @@ def __init__(self):
         self.show_all()
         self._set_wait_cursor()
 
+        self._install_screen._pgl.hide()
```

Comment:
Well spotted, thank you @tombettany! I forgot I did it the dirty way, when initially I had everything in `views/install.py` which is much better! I'll move things there which will make things cleaner in `install_window.py`. I will however need to call the `hide` and `show` methods on the label, because at the moment I do not know whether I can add more things to the `progress_grid` after creation/showing. I'll leave it as a TODO.